internal/uvm/virtual_device.go

```diff
@@ -14,55 +14,104 @@ import (
 // VPCIDevice represents a vpci device. Holds its guid and a handle to the uvm it
 // belongs to.
 type VPCIDevice struct {
```

Comment:
Can you add comments on `VMBusGUID` and `deviceInstanceID` explaining how they are different and what they're used for?